### PR TITLE
[FIX] Toolbar: properly display image UI

### PR DESCRIPTION
**Pull request by Zinston**

### CODE REVIEW

File: packages/plugin-dom-layout/src/ActionableGroupDomObjectRenderer.ts [hunk 1]
```diff
@@ -36,4 +53,27 @@ export class ActionableGroupDomObjectRenderer extends NodeRenderer<DomObject> {
...
     }
+    /**
+     * Update option rendering after the command if the value of visible
+     * changed.
+     *
+     * @param actionable
+     * @param element
+     */
+    protected _updateActionables(): void {
+        for (const [actionable, element] of this.actionableNodes) {
```
Comment (dmo-odoo) [inline packages/plugin-dom-layout/src/ActionableGroupDomObjectRenderer.ts:64]: I don't understand. Why do we have two "_updateActionables" functions?

File: packages/plugin-odoo/src/Odoo.ts [hunk 2]
```diff
@@ -60,12 +61,9 @@ const imageClassesLabels = {
  * If more images are within the range, return undefined.
  */
 function getSingleImage(range: VRange): ImageNode | undefined {
-    const next = range.start.nextLeaf();
-    if (next instanceof ImageNode) {
-        const prev = range.end.previousLeaf();
-        if (prev === next) {
-            return next;
-        }
+    const selected = range.selectedNodes(node => node instanceof AtomicNode);
```
Comment (dmo-odoo) [inline packages/plugin-odoo/src/Odoo.ts:64]: I think this function was previously refactored by @Gorash for performance reasons. Further checking is needed to ensure your version is not reverting back to a less performant state.

File: packages/plugin-dom-layout/src/ActionableGroupDomObjectRenderer.ts (hunk 1 above)
Comment (Zinston) [reply packages/plugin-dom-layout/src/ActionableGroupDomObjectRenderer.ts:64]: We have three :sunglasses:
One for Actionable, one for ActionableGroup, and one for ActionableGroupSelectItem.

File: packages/plugin-odoo/src/Odoo.ts (hunk 2 above)
Comment (Zinston) [reply packages/plugin-odoo/src/Odoo.ts:64]: @Gorash @Gorash @Gorash

Comment (Gorash) [reply packages/plugin-odoo/src/Odoo.ts:64]: no!
don't use selectedNodes in getSingleImage, it's called fto display button too.

File: packages/plugin-dom-layout/src/ActionableGroupDomObjectRenderer.ts (hunk 1 above)
Comment (dmo-odoo) [reply packages/plugin-dom-layout/src/ActionableGroupDomObjectRenderer.ts:64]: But... only the one of Actionable shoulb be called `_updateActionables`, right ? Shouldn't the others be `_updateActionableGroupItems` and `_updateActionableGroupeSelectItems` ?

**Comment by Zinston** [comment]
yes sure

**Comment by dmo-odoo** [comment]
@Zinston This PR has conflicts.

**Pull request merged**